models/grafana_helper.go

```diff
@@ -203,6 +215,21 @@ func (g *GrafanaClient) ProcessBoard(ctx context.Context, c *sdk.Client, board *
 					}
 				}
 				grafBoard.Panels = append(grafBoard.Panels, p1)
+			} else if p1.OfType != sdk.TextType && p1.OfType != sdk.TableType && p1.Type == "row" && len(p1.Panels) > 0 {
+				for _, p2 := range p1.Panels {
```

Comment:
A couple more comments through this set of nested for loops / if conditions might be nice.